tools/bionano/bionano_scaffold_edited.xml

```diff
@@ -76,7 +75,7 @@
         <flag attr="RAmem" val0="3" val1="1"/>
     </global>
     <fasta2cmap>
-        <flag attr="enzyme" val0="CTTAAG" display="Enzyme" group="FASTA to CMAP digestion" description="Define single enzyme for in-silico FASTA to CMAP digestion. Avalible enzymes: BspQI, BbvCI, BsmI, BsrDI, BssSI, DLE1."/>
+        <flag attr="enzyme" val0="${configuration_options.enzyme}" display="Enzyme" group="FASTA to CMAP digestion" description="Define single enzyme for in-silico FASTA to CMAP digestion. Avalible enzymes: BspQI, BbvCI, BsmI, BsrDI, BssSI, DLE1."/>
```

Comment:
This is not possible because the $configuration_options.enzyme is only set when  $configuration_options == 'VGP'

Reply:
Than add an if-condition.